### PR TITLE
Fix postpone appointment

### PR DESCRIPTION
**Pull request by geeky-abhishek**

### CODE REVIEW

File: src/view/reminder/commons/reminder-time-options.ts [hunk 1]
```diff
@@ -3,79 +3,196 @@
...
+	click?: () => void;
+	value: number;
+	type?: string;
+}>;
+export const useGetReminderItems = (
+	t: TFunction,
+	setSnooze: (arg1: number, arg2?: boolean) => void,
+	event: EventType
+): ReturnType => {
+	const diff = moment(event?.resource?.alarmData?.[0]?.alarmInstStart).diff(moment(), 'seconds');
```
Comment (nubsthead) [inline src/view/reminder/commons/reminder-time-options.ts]: this could be memoized too

File: src/view/reminder/commons/reminder-time-options.ts [hunk 2]
```diff
@@ -3,79 +3,196 @@
...
+
+type ReturnType = Array<{
+	id: string;
+	label?: string;
+	click?: () => void;
+	value: number;
+	type?: string;
+}>;
+export const useGetReminderItems = (
+	t: TFunction,
```
Comment (nubsthead) [inline src/view/reminder/commons/reminder-time-options.ts]: Can "t" be instantiated inside the hook?

File: src/view/reminder/commons/reminder-time-options.ts (hunk 1 above)
Comment (geeky-abhishek) [reply src/view/reminder/commons/reminder-time-options.ts]: ya sorry it just skip out of mind.Thanks for pointing it out

File: src/view/reminder/commons/reminder-time-options.ts (hunk 2 above)
Comment (geeky-abhishek) [reply src/view/reminder/commons/reminder-time-options.ts]: Yes it can,I'll update it

**Pull request closed without merging**